src/shared/trim-and-dedent.ts

```diff
@@ -19,16 +19,14 @@ function dedentStr(str: string, indent: number) {
  * params verbatim.
  */
 function dedent(strings: string[], ...params: any[]) {
-  // Match the smallest indentation among all strings
-  const indents = strings
-    .map(str => {
-      const match = str.match(/^[ \t]*(?=\S)/gm);
-      return match ? Math.min(...match.map(el => el.length)) : -1;
-    })
-    // Exclude lines where indentation could not be matched
-    .filter(num => num > -1);
```

Comment:
This was incorrectly treating every string as a line, while there can be multiple strings that belong to the same line.

With this logic, strings that are not the first in a line return an indent of `0`, resulting in no dedenting at all.